Files/MultilingualResources/Files.zh-Hans.xlf

```diff
@@ -1353,603 +1347,603 @@
-          <target state="new">This device is currently in use. Close any programs, windows or tabs that might be using the device, and then try again.</target>
+          <target state="translated">该设备正在使用中。关闭可能使用该设备的所有程序，窗口或标签页，然后重试。</target>
         </trans-unit>
         <trans-unit id="SideBarEjectDevice.Text" translate="yes" xml:space="preserve">
           <source>Eject</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">弹出</target>
+          <target state="translated">弹出</target>
         </trans-unit>
         <trans-unit id="HorizontalMultitaskingControlDuplicateTab.Text" translate="yes" xml:space="preserve">
           <source>Duplicate tab</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">重复打开选项卡</target>
+          <target state="translated">复制选项卡</target>
         </trans-unit>
         <trans-unit id="HorizontalMultitaskingControlMoveTabToNewWindow.Text" translate="yes" xml:space="preserve">
           <source>Move tab to new window</source>
-          <target state="new">Move tab to new window</target>
+          <target state="translated">将标签页移动到新窗口</target>
         </trans-unit>
         <trans-unit id="HorizontalMultitaskingControlNewTab.Text" translate="yes" xml:space="preserve">
           <source>New tab</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">新选项卡</target>
+          <target state="translated">新建标签页</target>
         </trans-unit>
         <trans-unit id="NavSearchButton.AutomationProperties.Name" translate="yes" xml:space="preserve">
           <source>Search</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">搜索</target>
+          <target state="translated">搜索</target>
         </trans-unit>
         <trans-unit id="PropertiesDialogReadOnly.Text" translate="yes" xml:space="preserve">
           <source>Read-only</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">只读</target>
+          <target state="translated">只读</target>
         </trans-unit>
         <trans-unit id="PropertiesDialogHidden.Text" translate="yes" xml:space="preserve">
           <source>Hidden</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">隐藏</target>
+          <target state="translated">隐藏</target>
         </trans-unit>
         <trans-unit id="PropertyRatingText" translate="yes" xml:space="preserve">
           <source>Rating Text</source>
-          <target state="new">Rating Text</target>
+          <target state="translated">分级文本</target>
         </trans-unit>
         <trans-unit id="PropertyItemFolderPathDisplay" translate="yes" xml:space="preserve">
           <source>Item Folder Path Display</source>
-          <target state="new">Item Folder Path Display</target>
+          <target state="translated">项目文件夹路径显示</target>
```

Comment:
What does the `Display` mean here?

Reply:
@linsui It means that this path is meant to be read by the user, and not parsed programmatically. It says that because the names were generated from its property system name, which is "System.ItemFolderPathDisplay". It can probably be ignored/removed.